src/sharedTypes.ts

```diff
@@ -1,14 +1,21 @@
-// TODO: Define transcription schema (this is just a mockup)
-
-import Transcription from 'sharedTypes/Transcription';
-
 export interface ProjectBase {
   schemaVersion: number;
   name: string;
   filePath: string;
   transcription: Transcription | null;
 }
 
+export interface Transcription {
```

Comment:
Wondering what the rationale is for having all of our shared types in one file instead of having a folder of individual files?

Reply:
When this gets bigger we might want to split it - for now I couldn't find a way to split it and have an index so that imports all happen from the same file, so I felt the tradeoff of including some extra stuff in here was fine. I'm sure there's a way to do it, but couldn't find it in like 10 mins of experimentation

The file is like 30 lines, it's perfectly fine to have the types together for now